Makefile missing install/uninstall targets

## Expected Behavior
`make install` --> install fdstat
`make uninstall` --> uninstall fdstat 

## Actual Behavior
`make: *** No rule to make target 'install'.  Stop.`
`make: *** No rule to make target 'uninstall'.  Stop.`

## Steps to Reproduce the Problem

  1. `make install`
  1. `make uninstall`

## System information
  - Version: 0.1.1

--- a/Makefile
+++ b/Makefile
@@ -2,8 +2,8 @@ CC = clang
 CFLAGS = -Wall -pedantic -std=c99
 
 OBJECTS = $(filter-out main.o,$(patsubst %.c,%.o,$(wildcard *.c)))
-# OTHER = strconst.h
 BIN = fdstat
+PREFIX = /usr/bin
 
 ifdef DEBUG
 CFLAGS += -DDEBUG -g
@@ -18,6 +18,11 @@ $(BIN): main.c $(OBJECTS)
 %.o: %.c %.h $(OTHER)
 	$(CC) -c $(CFLAGS) $< -o $@
 
+install: $(BIN)
+	install -D -m 0755 $(BIN) $(DESTDIR)$(PREFIX)/$(BIN)
+
+uninstall:
+	rm -f $(DESTDIR)$(PREFIX)/$(BIN)
 
 clean:
 	rm -fr $(BIN) *.o

--- a/debian/files
+++ b/debian/files
@@ -1,2 +1,2 @@
-fdstat_0.1.0-1_amd64.buildinfo admin optional
-fdstat_0.1.0-1_amd64.deb admin optional
+fdstat_0.1.1-1_amd64.buildinfo admin optional
+fdstat_0.1.1-1_amd64.deb admin optional

